lib/std/fs/get_app_data_dir.zig

```diff
@@ -56,6 +56,13 @@ pub fn getAppDataDir(allocator: *mem.Allocator, appname: []const u8) GetAppDataD
             };
             return fs.path.join(allocator, &[_][]const u8{ home_dir, ".local", "share", appname });
         },
+        .haiku => {
+            const home_dir = os.getenv("HOME") orelse {
```

Comment:
Haiku applications should store their data in `~/config/settings`, which you can determine by using the [`find_directory` API](https://www.haiku-os.org/docs/api/FindDirectory_8h.html).

Reply:
see 3 lines below where they are creating the path `~/config/settings`

But yes, it looks like `find_directory` would be the more official API

Reply:
i have added in a very bare-bones implementation using the find_directory() as suggested.